### PR TITLE
Fixes for the SQL Parser

### PR DESCRIPTION
**Pull request by dbMundada**
- Add space to avoid concatenation of column after EOF
- Add SqlCleaner class to remove the comments
- Support Drop query parsing
- Support SetOperationList parsing

### CODE REVIEW

File: src/main/scala/ai/privado/utility/SQLParser.scala [hunk 1]
```diff
@@ -1,65 +1,130 @@
...
+              Some(List(("SELECT", tableName, columns)))
+
+            case setOpList: SetOperationList =>
+              val selectStmts = setOpList.getSelects.toList
+              val tableNameColumnListMap = selectStmts.map { stmt =>
+                val plainSelect = stmt.asInstanceOf[PlainSelect]
+                val tableName   = plainSelect.getFromItem.toString
+                (
+                  tableName,
+                  plainSelect.getSelectItems.flatMap { case item: SelectItem =>
```
Comment (khemrajrathore) [inline src/main/scala/ai/privado/utility/SQLParser.scala:74]: The code block from LNo 74 to LNo 79 is repeated above as well, Let's have a function for this

File: src/main/scala/ai/privado/utility/SQLParser.scala [hunk 2]
```diff
@@ -1,65 +1,130 @@
...
+                  item.toString match {
+                    case f: String if f.contains("(") =>
+                      val function = CCJSqlParserUtil.parseExpression(f).asInstanceOf[Function]
+                      function.getParameters.getExpressions.toString.map(_.toString)
+                    case _ => List(item.toString)
+                  }
+                }.toList
+              Some(List(("SELECT", tableName, columns)))
+
+            case setOpList: SetOperationList =>
```
Comment (khemrajrathore) [inline src/main/scala/ai/privado/utility/SQLParser.scala:67]: Add a sample query to make this case understandable

File: src/main/scala/ai/privado/passes/SQLParser.scala [hunk 3]
```diff
@@ -76,34 +76,41 @@ class SQLParser(cpg: Cpg, projectRoot: String) extends ForkJoinParallelCpgPass[S
...
     sqlQueries
-      .flatMap(queryWthLine => {
-
+      .map(queryWthLine => {
         val query           = queryWthLine._1
         val queryLineNumber = queryWthLine._2
         try {
           SQLParser.parseSqlQuery(query) match {
-            case Some((queryName, tableName, columns)) =>
+            case Some(tmp) =>
```
Comment (khemrajrathore) [inline src/main/scala/ai/privado/passes/SQLParser.scala:84]: rename this variable to something like `parsedQueryList`

File: src/main/scala/ai/privado/passes/SQLParser.scala [hunk 4]
```diff
@@ -76,34 +76,41 @@ class SQLParser(cpg: Cpg, projectRoot: String) extends ForkJoinParallelCpgPass[S
...
-                  .code(query)
-                  .name(tableName)
-                  .fullName(query)
-                  .value(columns.mkString(","))
-                  .lineNumber(queryLineNumber)
-              builder.addNode(sqlQueryNode)
-              Some(sqlQueryNode)
-            case None => None
+              val res = tmp.map { case (_, tableName, columns) =>
+                val columnList = columns.mkString(", ")
```
Comment (khemrajrathore) [inline src/main/scala/ai/privado/passes/SQLParser.scala:89]: Any specific reason to change the mkString from `,` to `, `?

Comment (dbMundada) [reply src/main/scala/ai/privado/passes/SQLParser.scala]: Updated

**Pull request merged**